purchases/src/main/kotlin/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -516,6 +521,7 @@ internal class Backend(
                     body,
                     postFieldsToSign = null,
                     backendHelper.authenticationHeaders,
+                    fallbackBaseURLs = appConfig.fallbackBaseURLs,
```

Comment:
Just thinking to myself but should we even pass these fallback URLs for non-supported endpoints? I guess we are handling that separately though so it might be ok to do it like this so we don't forget on new endpoints, and handle in the `Endpoint` as you're doing :)

Reply:
Yeah, I had this same dilemma about this... honestly, I don't know what the best option is. That said, I wanted to avoid duplicating the logic and, most importantly, as you say, I didn't want to forget adding this on new endpoints. 

My first approach was to have the `fallbackBaseURLs` parameter default to `appConfig.fallbackBaseURLs`, but that led to some issues (in unit tests, IIRC).